js/ui/camera.js

```diff
@@ -29,7 +29,8 @@ const Evented = require('../util/evented');
  *
  * @typedef {Object} AnimationOptions
  * @property {number} duration The animation's duration, measured in milliseconds.
- * @property {Function} easing The animation's easing function.
+ * @property {Function} easing A function taking a time in the range 0..1 and returning a number where 0 is 
+ *                             the initial state and 1 is the final state.
```

Comment:
What makes you think that the function is expected to be nondecreasing? (I'm ok with using math class language!)